src/registry/datetime/validationSchema.ts

```diff
@@ -0,0 +1,87 @@
+import type {DateTimeComponentSchema} from '@open-formulieren/types';
+import {isValid, parseISO} from 'date-fns';
+import {defineMessage} from 'react-intl';
+import {z} from 'zod';
+
+import type {GetValidationSchema} from '@/registry/types';
+
+const DATETIME_INVALID_MESSAGE = defineMessage({
+  description: 'Invalid input validation error for datetime field',
+  defaultMessage: `The datetime must consist of a date and a time stamp (24-hour format without
+  seconds), separated by a space (e.g. 10/30/2025 12:34).`,
+});
+
+const DATETIME_GREATER_THAN_MAX_DATE_MESSAGE = defineMessage({
+  description: 'Validation error for datetime greater than maximum date.',
+  defaultMessage: 'The datetime must be earlier than or equal to {max}.',
+});
+
+const DATETIME_LESS_THAN_MIN_DATE_MESSAGE = defineMessage({
+  description: 'Validation error for datetime less than minimum date.',
+  defaultMessage: 'The datetime must be later than or equal to {min}.',
+});
+
+const getValidationSchema: GetValidationSchema<DateTimeComponentSchema> = (
+  componentDefinition,
+  intl
+) => {
+  const {key, validate = {}, datePicker} = componentDefinition;
+  const {required} = validate;
+  // In the backend, we set/grab the min and max dates from the `datePicker` property, so we also
+  // need to do this here. Once we swapped formio for our own renderer - and also implemented
+  // support in the form builder for choosing which widget to use - the min and max dates should be
+  // moved to either the `validate` or `openForms` prop, probably.
+  const minDate = datePicker?.minDate;
+  const maxDate = datePicker?.maxDate;
+
+  let dateSchema = z.coerce.date();
+  if (minDate) {
+    const minDateString = intl
+      .formatDate(minDate, {
+        year: 'numeric',
+        month: 'numeric',
+        day: 'numeric',
+        hour: 'numeric',
+        minute: '2-digit',
+        hour12: false,
+      })
+      .replace(',', '');
```

Comment:
what exactly produces a comma?

Reply:
okay - it bothers me a bit that we use general purpose `intl` library which uses the browser native `Intl` APIs under the hood and then still end up tweaking the format manually :thinking: 

Since this is purely informational in the validation error message, I think we can keep the comma.